### PR TITLE
Update Ammo.js library and AmmoJS plugin to support ellipsoid

### PR DESCRIPTION
**Pull request by CedricGuillemet**
Update Ammo.js library and AmmoJS plugin to support ellipsoid

### CODE REVIEW

File: src/Physics/Plugins/ammoJSPlugin.ts [hunk 1]
```diff
@@ -900,7 +900,16 @@ export class AmmoJSPlugin implements IPhysicsEnginePlugin {
 
         switch (impostor.type) {
             case PhysicsImpostor.SphereImpostor:
-                returnValue = new Ammo.btSphereShape(extendSize.x / 2);
+                // Is there a better way to compare floats number? With an epsylon or with a Math function
+                if (extendSize.x == extendSize.y && extendSize.x == extendSize.z) {
```
Comment (deltakosh) [inline src/Physics/Plugins/ammoJSPlugin.ts]:
```suggestion
                if (extendSize.x === extendSize.y && extendSize.x === extendSize.z) {
```

**Comment by deltakosh** [comment]
There is just a linting issue

File: src/Physics/Plugins/ammoJSPlugin.ts [hunk 2]
```diff
@@ -900,7 +900,16 @@ export class AmmoJSPlugin implements IPhysicsEnginePlugin {
 
         switch (impostor.type) {
             case PhysicsImpostor.SphereImpostor:
-                returnValue = new Ammo.btSphereShape(extendSize.x / 2);
+                // Is there a better way to compare floats number? With an epsylon or with a Math function
```
Comment (bghgary) [inline src/Physics/Plugins/ammoJSPlugin.ts]: You can use Scalar.WithinEpsilon

Comment (tom-berend) [reply src/Physics/Plugins/ammoJSPlugin.ts]: tested this fix with my spinning ellipsoids, works perfectly.

**Comment by deltakosh** [comment]
Still need a linting fix

**Pull request merged**